js/main.js

```diff
@@ -45,7 +33,50 @@ if (shouldQuit) {
 app.on('ready', getUrlAndOpenMainWindow);
 
 function getUrlAndOpenMainWindow() {
-    // for dev env allow passing url argument
+    let installMode = false;
+    
+    process.argv.some((val) => {
+        let flag = '--install';
+        if (val === flag) {
+            installMode = true;
+            getConfigField('launchOnStartup')
+            .then(setStartup);
+        }
+
+        return false;
+    });
+
+    if (installMode === false){
+        openMainWindow();
+    }
+}
+
+function setStartup(lStartup){
+    if (lStartup === true){
+        symphonyAutoLauncher.isEnabled()
+        .then(function(isEnabled){
+            if(isEnabled){
```

Comment:
Does it make sense to use a code formatter (with a common rules profile) so that it is consistent across the board? 

This might help -> https://github.com/millermedeiros/esformatter

It supports plugins for VSCode, Atom, Vim and Sublime, so, looks pretty good really.